src/components/home/elements/DepositToken.vue

```diff
@@ -0,0 +1,634 @@
+          <TokenConfirm :address="address"
+                        :amount=inputAmount(amount)
+                        :fee="fee"
+                        :contract-id="contractId"
+                        :tx-type="'Deposit Token to Contract'">
+          </TokenConfirm>
+          <b-row>
+            <b-col class="col-lef">
+              <b-button
+                class="btn-back"
+                block
+                variant="light"
+                size="lg"
+                @click="coldPrevPage">Back
+              </b-button>
+            </b-col>
+            <b-col class="col-rit">
+              <b-button
+                block
+                class="btn-confirm"
+                variant="warning"
+                size="lg"
+                @click="coldNextPage">Confirm
+              </b-button>
+            </b-col>
+          </b-row>
+        </b-container>
+        <b-container v-if="coldPageId===3"
+                     class="text-left">
+          <ColdSignature :data-object="dataObject"
+                         :qr-total-page="1"
+                         v-if="coldPageId===3"
+                         @get-signature="getSignature"
+                         @next-page="coldNextPage"
+                         @prev-page="coldPrevPage"></ColdSignature>
+        </b-container>
+        <b-container v-show="coldPageId===4">
+          <TokenConfirm :address="address"
+                        :amount=inputAmount(amount)
+                        :fee="fee"
+                        :contract-id="contractId"
+                        :tx-type="'Deposit Token to Contract'">
+          </TokenConfirm>
+          <p v-show="sendError">Sorry, transaction send failed! {{ errorMessage }}</p>
+          <b-row>
+            <b-col class="col-lef">
+              <b-button
+                class="btn-back"
+                block
+                variant="light"
+                size="lg"
+                @click="coldPrevPage">Back
+              </b-button>
+            </b-col>
+            <b-col class="col-rit">
+              <b-button
+                block
+                class="btn-confirm"
+                variant="warning"
+                size="lg"
+                @click="sendData('coldWallet')">Confirm
+              </b-button>
+            </b-col>
+          </b-row>
+        </b-container>
+        <b-container v-show="coldPageId===5">
+          <TokenSuccess class="tokenSucced"
+                        :address="address"
+                        :amount=inputAmount(amount)
+                        :fee="fee"
+                        :contract-id="contractId"
+                        :tx-type="'Deposit Token to Contract'">
+          </TokenSuccess>
+          <b-button variant="warning"
+                    block
+                    size="lg"
+                    @click="endSend">OK
+          </b-button>
+        </b-container>
+      </b-tab>
+    </b-tabs>
+  </b-modal>
+</template>
```

Comment:
Abstract these similar html to one component. And reuse the component with `v-slot`

Reply:
the next step refactoring wallet pr to solve this problem